### PR TITLE
Relation

### PR DESCRIPTION
**Pull request by sebastianbagya**
Finalize Relation

### CODE REVIEW

File: src/test/java/com/blibli/future/controller/AttendanceControllerTest.java [hunk 1]
```diff
@@ -110,57 +107,57 @@ public void uploadAttendanceFileTest_UnableToParse() throws Exception {
...
-        		MockMvcRequestBuilders.post("/employees/shift")
-                .contentType(MediaType.APPLICATION_JSON)
-                .content(employeeShiftVo)
-        		)
-        .andExpect(MockMvcResultMatchers.status().isOk());
-
-        Mockito.verify(converterService).map(employeeShiftVoMock, EmployeeShift.class);
-        Mockito.verify(employeeShiftingService).processShifting(employeeShiftMock);
-    }
+//    @Test
```
Comment (riencus) [inline src/test/java/com/blibli/future/controller/AttendanceControllerTest.java:110]: Why this test is commented?

File: src/main/java/com/blibli/future/controller/AttendanceController.java [hunk 2]
```diff
@@ -101,38 +102,37 @@ public ResponseEntity employeeTappingUpdate(@RequestParam("type") String type, @
         return new ResponseEntity(employeeTapGetted, HttpStatus.BAD_REQUEST);
```
Comment (riencus) [inline src/main/java/com/blibli/future/controller/AttendanceController.java:102]: I'm sorry this one really bothers me.
The past participle of "get" is "got/gotten" not "getted" 
"getted" is a non standard form [https://en.wiktionary.org/wiki/getted](url)

**Review by riencus — COMMENTED** [review]
Just some question and other little things

File: src/test/java/com/blibli/future/controller/AttendanceControllerTest.java (hunk 1 above)
Comment (sebastianbagya) [reply src/test/java/com/blibli/future/controller/AttendanceControllerTest.java:110]: ah, forgot to remake the test

File: src/main/java/com/blibli/future/controller/AttendanceController.java (hunk 2 above)
Comment (sebastianbagya) [reply src/main/java/com/blibli/future/controller/AttendanceController.java:102]: ok, thanks

**Pull request merged**